packages/dashboard-v2/gatsby-ssr.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "@fontsource/sora/300.css"; // light
+import "@fontsource/sora/400.css"; // normal
+import "@fontsource/sora/500.css"; // medium
+import "@fontsource/sora/600.css"; // semibold
+import "@fontsource/source-sans-pro/400.css"; // normal
+import "@fontsource/source-sans-pro/600.css"; // semibold
+import "./src/styles/global.css";
+
+export function wrapPageElement({ element, props }) {
+  const Layout = element.type.Layout ?? React.Fragment
+  return <Layout {...props}>{element}</Layout>
+}
```

Comment:
we're not doing ssr because our pages will be hosted statically, we can skip this file

Reply:
I initially added it because I had issues with first render on development server, but it looks like it's no longer needed (probably got resolved by clearing gatsby's `.cache` 🤔 )

Removed in 0c77705